contrib/dist/linux/README

```diff
@@ -40,42 +40,50 @@ The folowing arguments could be used to affect script behaviour.
 Please, do NOT set the same settings with parameters and config vars.
 
 -b
-   If you specify this option, all-in-one binary RPM would be created
-   instead of source RPM (SRPM). This option is necessary for other
-   parameters to work, as they won't be influence SRPM package.
+   If you specify this option, only all-in-one binary RPM would be build.
+   By default, only source RPM (SRPM) is build. This option is necessary
+   for all other parameters to work, as they work only with binary RPM.
```

Comment:
I apologize, but the English language is totally terrible -- it's full of special case grammar rules, especially in this paragraph.  Here's probably how it should read:

> If you specify this option, only the all-in-one binary RPM will be built.  By default, only the source RPM (SRPM) is built.  Other parameters that affect the all-in-one binary RPM will be ignored unless this option is specified.